### PR TITLE
Create wheels.js

### PR DESCRIPTION
**Pull request by BandiDuhHero**
Don't merge yet obviously, Idk you guys' economy functions so replace mine with them, to find them just search in ctrl + f Economy and Db nothingelse is reliant on my economy in there

### CODE REVIEW

**Comment by SilverTactic** [comment]
<i>Idk you guys' economy functions so replace mine with them</i>
dude, that's your job lol
Also, if you need to correct something, amend/squash your commits(s)

**Comment by SilverTactic** [comment]
You still need to do two things:
1) Indent your code with tabs and not spaces
2) Squash your commits into one

File: chat-plugins/wheels.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    endDisplay() {
+        return '|raw|<div style="padding: 5px; background-color: #FFFFCC; color: #FFB41F; border: solid, 1px, #FFB41F; font-size: 15px;">The wheel has landed on: <em><strong>' + this.wheelValue + '</strong></em><br>Positive Numbers = Host wins<br /> Negative Numbers = Spinner wins</div>'
+    }
+ };
+ const wheels = {
+    scrubs: [-5,-4,-3,-2,-2,-1,-1,-1,-1,-1,-1,+1,+1,+3,+3,+3,+3,+3,+3,+5],
+    wealth: [-200,-100,-100,-50,-50,-25,-25,-25,+50,+200,+200,+300,+300,+400],
+    diversity: [-40,-30,-20,-10,-15,-4,-2,-1,+2,+10,+30,+50,+50,+50],
+    balance: [-30,-20,-10,-5,-5,-5,-5,-2,-1,+3,+3,+15,+15,+30,+30,+30],
+    death; [-50,-5,-5,-1,-1,-1,-1,-1,-1,-1,-1,+1,+1,+1,+1,+10,+10,+50]
```
Comment (panpawn) [inline chat-plugins/wheels.js]: syntax error, this shouldn't be a semi colon

File: chat-plugins/wheels.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                '/wheel join/spin/start - Joins the wheel game. <br />' +
+                '/wheel end - Ends the current wheel game <br />');
+        },
+        list: function(target, room, user) {
+            if (!this.runBroadcast()) return;
+            this.sendReplyBox('<center>Wheels of Misfortune :]</center><hr />' +
+                'Wheel of Scrubs: -5, -4, -3, -2, -2, -1, -1, -1, -1, -1, -1, +1, +1, +3, +3, +3, +3, +3, +3, +5<br />' +
+                'Wheel of Balance: -30, -20, -10, -5, -5, -5, -5, -2, -1, +3, +3, +15, +15, +30, +30, +30<br />' +
+                'Wheel of Diversity: -40, -30, -20, -10, -15, -4, -2, -1, +2, +10, +30, +50, +50, +50<br />' +
+                'Wheel of Wealth: -200, -100, -100, -50, -50, -25, -25, -25, +50, +200, +200, +300, +300, +400<br />'
```
Comment (panpawn) [inline chat-plugins/wheels.js]: forgot + at the end of this line

File: chat-plugins/wheels.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+            Economy.readMoney(user.userid, money => {
+            	if (money < Math.abs(wheels[targets[0]][wheels[targets[0]].length]) * Number(targets[1])) return this.errorReply('You do not have enough bucks to create this game.');
+            	if(!checkMoney(targets[1])) return this.errorReply('Multipliers ' + isMoney(targets[1]) + '.');
+            	room.wheel = new Wheel(parseInt(targets[1]), user.userid, wheels[targets[0]], spinWheel(wheels[targets[0]]));
+            	room.add(room.wheel.initDisplay(target)); //make look fancy later
+            	room.add('|raw|<b><font color="blue">' + user.name + ' is hosting the wheel.</font></b>');
+            	room.update();
+            	Economy.writeMoney(user.userid, Math.abs(room.wheel.wheel[room.wheel.wheel.length]) * room.wheel.wheelMulti * -1);
+            });
+        },                                                                          ///should parse the final number
```
Comment (panpawn) [inline chat-plugins/wheels.js]: what's this comment doing so far away, did it get a timeout? Kappa

Comment (BandiDuhHero) [inline chat-plugins/wheels.js]: kyvn added that comment when we were first developing this my bad for not just removing in the first place

**Comment by SilverTactic** [comment]
In the future, syntax check your code using http://esprima.org/demo/validate.html
Or even better, test every update of yours :P

**Comment by panpawn** [comment]
doing just ``/animal`` makes the help command show up twice

**Comment by jd4564** [comment]
Merged in be7cbd532aed28afc27408cd4efaa1d8dd7d01bb

**Pull request closed without merging**